EAD-Session-3-Assignment/WebContent/addStudentDetails.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Add Student Details</title>
+</head>
+<body align="center">
+    <h2>
+        Add student Details
+    </h2>
+    <form action="AddStudentDetails" method="GET" >
+        <table align="center">
+            <tr>
+                <td>
+                    <span>First Name : </span>
```

Comment:
If you are not using any css and style for the text then do need to use <span> tag. You can use <p></p> tag. Otherwise write the simply text here, no need to use any tag.

Reply:
Used this because someone told me that it is best practice. If anytime we want to change the text dynamically, we will do it directly through js.